porter/v1/agent_app_event_types.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
```

Comment:
This is one of the two major changes

Reply:
Ignore these lol, there were a lot of small changes in every file before I rebased on main. Put these to highlight the main changes.